tracer-core/src/main/java/org/zalando/tracer/MDCSpanObserver.java

```diff
@@ -0,0 +1,61 @@
+package org.zalando.tracer;
+
+import io.opentracing.contrib.api.SpanData;
+import io.opentracing.contrib.api.SpanObserver;
+import io.opentracing.contrib.api.TracerObserver;
+import org.slf4j.MDC;
+
+import java.util.Map;
+
+import static org.zalando.tracer.Flow.Logging.FLOW_ID;
+import static org.zalando.tracer.Flow.Logging.SPAN_ID;
+import static org.zalando.tracer.Flow.Logging.TRACE_ID;
+
+/**
+ * %equals{%X{flow_id}}{}{%X{trace_id}}
+ */
+public final class MDCSpanObserver implements TracerObserver, SpanObserver {
+
+    @Override
+    public SpanObserver onStart(final SpanData data) {
+        // TODO https://github.com/opentracing-contrib/java-api-extensions/issues/17
```

Comment:
This is kind of a blocker.

Reply:
https://github.com/opentracing-contrib/java-api-extensions/issues/17

Reply:
via https://github.com/opentracing/opentracing-java/issues/314

Reply:
First one is done